autoid_service/autoid.go

```diff
@@ -0,0 +1,32 @@
+package autoid
+
+import (
+	"context"
+
+	"github.com/pingcap/kvproto/pkg/autoid"
+	"github.com/pingcap/tidb/kv"
+	"google.golang.org/grpc"
+)
+
+type mockClient struct {
+}
+
+func (m *mockClient) AllocAutoID(ctx context.Context, in *autoid.AutoIDRequest, opts ...grpc.CallOption) (*autoid.AutoIDResponse, error) {
+	panic("TODO")
```

Comment:
The mock one is here to pass build (after split PRs).
The functionality would be available after another PR been merged https://github.com/pingcap/tidb/pull/38810/files#diff-617839a099f419c84781f44a2286d763780a446a701eac0a440c7c579378706cR259

So the reviewers can skip here for now.